pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -650,6 +701,17 @@ func createDynamicPoolHostConfig() []runtimeclient.Object {
 	return []runtimeclient.Object{&cm, &sec}
 }
 
+func createLocalHostConfig() []runtimeclient.Object {
+	cm := v1.ConfigMap{}
+	cm.Name = HostConfig
+	cm.Namespace = systemNamespace
+	cm.Labels = map[string]string{ConfigMapLabel: "hosts"}
+	cm.Data = map[string]string{
+		"local-platforms": "linux/arm64",
```

Comment:
This mapping for a local-platform is only for the test, right? We may choose to use a similar pattern for local, but we may define any value that we want? For example, I had considered `local/arm64` but am okay to change if needed.

Reply:
Yes, this is test data.

I don't think it would be wise for us to call it "local/anything", wither an architecture runs in a "local pod" or a "remote VM" is not something that we should explicitly expose to user pipeline code. 
Consider what will happen when we will migrate from amd64 to arm64 for our production clusters, the "local" architecture will change, but we wouldn't want all the users to have to change their pipelines to use the new clusters.